lib/views/pages/organization/profile_page.dart

```diff
@@ -171,7 +155,7 @@ class ProfilePage extends StatelessWidget {
                               }),
                           model.isCreator == null
```

Comment:
Nicely Done!
 I see where the condition for showing "leave organization" vs "organization settings" is getting a bit complicated and making the view seem complex. How about we can make the code even cleaner by placing the logic to determine which button should be displayed in a getter in the ViewModel and return a boolean (Don't forget to add documentation)

Something like:
```dart
model.showLeaveButton ? FunctionToShowLeaveButton() : FunctionToShowOrgSettingsButton()
```

And inside our ViewModel we can then do something similar to:

```dart
/// Logic to determine if Leave Organization Button should be displayed on the profilePage
/// The Show Leave Button would be displayed when {Reason button will be displayed here}
bool get showLeaveButton {
// Logic that returns true if Leave Organization button should be displayed
// else return false
}
```

Reply:
@TheHazeEffect 
Since the only logic behind which button to show is _if the user is the creator_, so I have used the boolean _isCreator_ instead of writing the _shouldShowLeaveButton_ function. I have written down the documentation and also simplified the code. I hope this works. Please review it and let me know if any change is required.
Thanks